cli/lambda-node-version-update.md

```diff
@@ -0,0 +1,34 @@
+# Node Version Update
+AWS Lambda announced that it will deprecate its runtime support for Node.js 6.10  on April 30, 2019[[ref]](https://docs.aws.amazon.com/lambda/latest/dg/runtime-support-policy.html),
+and it has started to support Node.js 8.10 [[ref]](https://aws.amazon.com/about-aws/whats-new/2018/04/aws-lambda-supports-nodejs/).
+
+The Amplify CLI code base has been updated to reflect this change, if you use Amplify CLI version 1.6.5 and above
+to create new aws resources, this does not concern you.
+
+However, if you have used previous versions of the Amplify CLI to create aws resources in the following categories, 
+you will need to manually update your project to avoid Node.js runtime issues with AWS Lambda. 
+- analytics 
+- auth
+- interactions
+
+Before you make the following manual changes, thoroughly back up your project. 
```

Comment:
thoroughly back up your project --> please make sure to back up your entire project

Reply:
changed.